packages/core/mesh/rpc/src/rpc.ts

```diff
@@ -539,6 +539,14 @@ export class RpcPeer {
   }
 }
 
+const DEBUG_LOG_ALL_CALLS = true;
```

Comment:
quick way to disable very noisy logs

Reply:
this seems like an anti pattern:

first, this will loose any logging metadata (line numbers, instance ids, etc.)
second, there's a way to filter logs by filename, perhaps a better way would be to improve those filters instead (e.g, by adding negative filers: `-debug:rpc.ts`)